descartes_core/src/cart_trajectory_pt.cpp

```diff
@@ -203,7 +211,78 @@ bool CartTrajectoryPt::getClosestJointPose(const std::vector<double> &seed_state
                                            const RobotModel &model,
                                            std::vector<double> &joint_pose) const
 {
-  NOT_IMPLEMENTED_ERR(false);
+  Eigen::Affine3d nominal_pose,candidate_pose;
```

Comment:
We should add an issue to address this incomplete method.  I believe it will only work in the simplest of cases.